### ISSUE TITLE
error runtime 

### ISSUE THREAD

**Issue by DiegoRODestudiante** [IIC2343/Syllabus-2024-2#61]
hola, en la primera función parece que me dan todos los resultados correctos pero me da este error y no se que hacer.
![image](https://github.com/user-attachments/assets/ab05a6e1-e471-4a63-981e-3f8d0d54b059)

**Comment by Geeermy** [comment]
Hola, por tercera vez: sigue el formato del título de _issues_ establecido en el README de este repositorio. Esperaré a que edites el título para contestarte.

**Comment by Geeermy** [comment]
@DiegoRODestudiante sí, gracias. Es importante para poder filtrar bien las _issues_.

Respondiendo a tu pregunta. Creo que necesitaré ver la línea de código 200 de tu programa. **No la mandes por aquí**, envíame un correo, por favor. Te sigo contestando por ahí.

**Comment by Geeermy** [comment]
PD: Para las personas curiosas, faltaba actualizar el archivo N-alumno.asm a la última versión.
